### ISSUE TITLE
Total accessibility can be viewed in the tSNE

### ISSUE THREAD

**Issue by jhc-at-home** [aertslab/SCope#239]
> To visualize multiple regions (e.g., a Topic, or an enhancer signature): the
> total accessibility can be viewed in the tSNE, similarly as a gene signature
> in SCope v1. For, the genomic view of the enhancer signature, the
> TOUCAN web viewer (developed by the BITS team) can be used.

**Comment by jhc-at-home** [comment]
This TOUCAN web viewer is [here](https://github.com/aertslab/ToucanJS)?

**Comment by KrisDavie** [comment]
> This TOUCAN web viewer is [here](https://github.com/aertslab/ToucanJS)?

Yeah, I asked internally and that seems like that's the most up to date version.

**Comment by jhc-at-home** [comment]
There is a version at http://dev.bits.vib.be:8888/toucan.js/ that is different (eyeballing). I've contacted Lukaz for corresponding source.
